src/components/panels/collapsible-panel/README.md

```diff
@@ -33,21 +33,22 @@ The component can be used in a controlled or uncontrolled manner. Please refer t
+| `onToggle`              | `func`   | (required only if controlled as in `isClosed` is passed) | -                     | -         | function to be triggered whenever the user clicks the top area to collapse the panel's content                                            |
+| `isClosed`              | `bool`   |                                                          | -                     | false     | Indicates if the panel's content should be collapsed or shown                                                                             |
+| `isSticky`              | `bool`   |                                                          | -                     | false     | Makes the panel's header sticky in regards to the page's scroll                                                                           |
+| `description`           | `String` |                                                          | -                     |           | If passed will be shown below the title as more information regarding the panel                                                           |
+| `isDisabled`            | `bool`   |                                                          | -                     | false     | Disables the panel and all interactions with it                                                                                           |
+| `children`              | `node`   |                                                          | -                     | -         | The actual content rendered inside the panel                                                                                              |
+| `hideExpansionControls` | `bool`   |                                                          | -                     | -         | Controls the visibility of the expansion controls on the left                                                                             |
+| `headerControls`        | `node`   |                                                          | -                     | -         | Controls at the top right part of the panel                                                                                               |
+| `tone`                  | `oneOf`  |                                                          | ['urgent', 'primary'] | 'primary' | -                                                                                                                                         |
+| `className`             | `bool`   |                                                          | -                     | -         | -                                                                                                                                         |
```

Comment:
Note in the scope of this PR obviously but this should go away so no custom CSS leaks into the component 😋

Reply:
Yeah, it would be more appropriate to create an issue for that.